eth/api_backend.go

```diff
@@ -68,10 +68,11 @@ func (b *EthAPIBackend) GetLatestEth1Data() (common.Hash, uint64) {
 
 func (b *EthAPIBackend) GetRollupContractAddresses() map[string]*common.Address {
 	return map[string]*common.Address{
-		"addressResolver":           &b.eth.syncService.AddressResolverAddress,
-		"canonicalTransactionChain": &b.eth.syncService.CanonicalTransactionChainAddress,
-		"sequencerDecompression":    &b.eth.syncService.SequencerDecompressionAddress,
-		"stateCommitmentChain":      &b.eth.syncService.StateCommitmentChainAddress,
+		"addressResolver":               &b.eth.syncService.AddressResolverAddress,
+		"canonicalTransactionChain":     &b.eth.syncService.CanonicalTransactionChainAddress,
+		"sequencerDecompression":        &b.eth.syncService.SequencerDecompressionAddress,
+		"stateCommitmentChain":          &b.eth.syncService.StateCommitmentChainAddress,
+		"l1CrossDomainMessengerAddress": &b.eth.syncService.L1CrossDomainMessengerAddress,
```

Comment:
Ah did you not include the `RollupAddressManagerOwnerAddress` here because you can just query it in the smart contract? Could add it but imo definitely not needed.

Just trying to think of things to catch cuz I didn't see anything worth changing! Woot!

Reply:
I didn't include it in this because its not something that is resolvable via the address resolver in the same way as the other ones. Its instead a public variable. Not super opinionated on this